lib/screens/create_region_screens/add_region_background_image.dart

```diff
@@ -17,69 +17,71 @@ class AddRegionBackgroundImage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return BlocConsumer<CreateRegionBloc, CreateRegionState>(
-        listenWhen: (previous, current) => current.pageCommand != null || previous.file != current.file,
-        listener: (context, state) {
-          if (state.pageCommand != null) {
-            final pageCommand = state.pageCommand;
+      listenWhen: (previous, current) => current.pageCommand != null || previous.file != current.file,
```

Comment:
|| previous.file != current.file is this needed?

Reply:
Changed for new pageCommand RemoveAuthenticationScreen